src/components/list.js

```diff
@@ -3,6 +3,51 @@ import PropTypes from 'prop-types';
 import { getStyles } from '../utils/base';
 import styled from 'react-emotion';
 
+// Is it okay to export from here?  Should they go in different files?
+export const bulletStyles = {
+  star: '\\2605',
+  classicCheck: '\\2713',
+  greenCheck: '\\2705',
+  arrow: '\\219d',
+  cross: '\\274C'
+};
+
+export const getBulletStyle = (bulletStyle, isListItemStyle) => {
+  if (bulletStyle) {
+    const content = bulletStyles[bulletStyle] || `\\${bulletStyle}`;
+
+    if (isListItemStyle === true) {
+      return [
+        { listStyleType: 'none' },
+        `&::before {
+          content: '${content}' !important;
```

Comment:
@ebrillhart I had to use `!important` as List element had higher specificity than ListItem and I wanted to overwrite List styling with ListItem styling in some cases. Mariano mentioned that in this scenario it will be ok to use it. Let me know what you think.